### PR TITLE
Updated status of serveral tests

### PR DESCRIPTION
**Pull request by wang3820**
Opened real PR for 2 tests:
https://github.com/jetty/jetty.project/pull/10885

Change status of 2 tests from "OD" to "NOD": verified with iDFlakies with detector type`random-class-method` and 100 random rounds. The logs are in my vm at `~/project/semanticvectors/`

### CODE REVIEW

**Comment by wang3820** [comment]
Not sure why there is a format error on the PR link yet. The link looks fine to me and it's valid. I'll look into the format checker.

**Comment by wang3820** [comment]
So https://github.com/eclipse/jetty.project no longer exists, the previous link goes directly to https://github.com/jetty/jetty.project and I made the PR against this project. This project is not on IDOFT and I do not think anybody is working on it. Should I just change the project link?

File: pr-data.csv [hunk 1]
```diff
@@ -4049,8 +4049,8 @@ https://github.com/logzio/sawmill,e493c2e279b8820c0fdc2c79ab760d6759eb5a4a,sawmi
 https://github.com/looly/hutool,91565d052d2771dd26547e79c619ee8c31497e8f,hutool-core,cn.hutool.core.io.FileTypeUtilTest.fileTypeUtilTest,ID;NIO,Skipped,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/67
 https://github.com/looly/hutool,c1642522bbf5adc70fb57517e46d5a921d9a5703,hutool-core,cn.hutool.core.util.EnumUtilTest.getFieldNamesTest,ID,Accepted,https://github.com/looly/hutool/pull/1270,
 https://github.com/looly/hutool,91565d052d2771dd26547e79c619ee8c31497e8f,hutool-cron,cn.hutool.cron.test.CronTest.matchAllTest,NDOD;NOD,,,
-https://github.com/Lucky-Dhakad/semanticvectors,cf2d883c5fafd84ca2530fb15e62ba399ebc7176,.,pitt.search.semanticvectors.integrationtests.RegressionTests.testBuildAndSearchComplexPermutationIndex,OD,,,
-https://github.com/Lucky-Dhakad/semanticvectors,cf2d883c5fafd84ca2530fb15e62ba399ebc7176,.,pitt.search.semanticvectors.vectors.ComplexVectorTest.testComplexOrthogonalize,OD,,,
+https://github.com/Lucky-Dhakad/semanticvectors,cf2d883c5fafd84ca2530fb15e62ba399ebc7176,.,pitt.search.semanticvectors.integrationtests.RegressionTests.testBuildAndSearchComplexPermutationIndex,NOD,,,
```
Comment (darko-marinov) [inline pr-data.csv:4052]: Why are these `NOD`?

File: pr-data.csv [hunk 2]
```diff
@@ -3156,8 +3156,8 @@ https://github.com/eclipse-ee4j/tyrus,d86e0cb0a4f26ba89daf8668137c156e57fea565,t
 https://github.com/eclipse-ee4j/tyrus,d86e0cb0a4f26ba89daf8668137c156e57fea565,tests/e2e/standard-config,org.glassfish.tyrus.test.standard_config.SessionTimeoutTest.testSessionTimeout,UD,,,
 https://github.com/eclipse-ee4j/tyrus,d86e0cb0a4f26ba89daf8668137c156e57fea565,tests/e2e/standard-config,org.glassfish.tyrus.test.standard_config.SessionTimeoutTest.testSessionTimeoutChanged,NOD,,,
 https://github.com/eclipse-ee4j/tyrus,d86e0cb0a4f26ba89daf8668137c156e57fea565,tests/servlet/autobahn-server,org.glassfish.tyrus.tests.servlet.autobahn.StandaloneServer.testServer,UD,,,
-https://github.com/eclipse/jetty.project,9cede68b355515e1cf8802fd1fcf67aa7547aad1,jetty-server,org.eclipse.jetty.server.ssl.SniSslConnectionFactoryTest.testSNIConnectNoWild,ID,,,
-https://github.com/eclipse/jetty.project,9cede68b355515e1cf8802fd1fcf67aa7547aad1,jetty-server,org.eclipse.jetty.server.ssl.SslConnectionFactoryTest.testSNIConnect,ID,,,
+https://github.com/eclipse/jetty.project,9cede68b355515e1cf8802fd1fcf67aa7547aad1,jetty-server,org.eclipse.jetty.server.ssl.SniSslConnectionFactoryTest.testSNIConnectNoWild,ID,Opened,https://github.com/jetty/jetty.project/pull/10885,
```
Comment (darko-marinov) [inline pr-data.csv:3159]: The checker failed. You need to make the project URL and PR URL in sync. Rename the project to `github.com/jetty/jetty.project` (and resort appropriately as per the IDoFT README).

File: pr-data.csv (hunk 1 above)
Comment (wang3820) [reply pr-data.csv:4052]: Yes. And I checked with iDFlakies too. With too few runs, sometimes it passes in all the orders, sometimes it only fail and never pass in certain order and get classified as OD, I checked with iDFlakies with detector as random-class-method and with 100 random rounds, and those two turns out to be NOD, in some order it passes and fails. The logs are in my vm.

**Pull request merged**